### PR TITLE
Test data migrations

### PR DESCRIPTION
**Pull request by seb879**
add a few test data in database

### CODE REVIEW

**Review by erabin05 — CHANGES_REQUESTED** [review]
- You miss all the ```exports.down``` function in your migrations. It supposed to do the opposite of what you do in ```exports.up``` 
- And by the way you can do all your changes in only one migration

File: migrations/20190605091052-Table-Keyword_With_Fk.js [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
-exports.setup = function (options, seedLink) {
+exports.setup = function(options, seedLink) {
   dbm = options.dbmigrate;
   type = dbm.dataType;
   seed = seedLink;
 };
 
-exports.up = function (db) {
+exports.up = function(db) {
   return db.createTable('keywordwithfk', {
```
Comment (dadtmt) [inline migrations/20190605091052-Table-Keyword_With_Fk.js]: not a proper name for a table

File: migrations/20190605091046-Table-Keywords.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+exports.up = function(db) {
+  return  db.createTable('keyword', {
+    id: {
+      type: 'int', primaryKey: true, autoIncrement: true, notNull: true,
+    },
+    name: { type: 'char', length: 80, notNull: true },
+    importance: { type: 'int', notNull: true },
+  })
+  .then(
+    function(result){
```
Comment (dadtmt) [inline migrations/20190605091046-Table-Keywords.js]: creating tables and filling tables should not be part of the same file . one day you will want to create the db without seeding it

**Review by dadtmt — CHANGES_REQUESTED** [review]

File: migrations/20190605091052-Table-Keyword_With_Fk.js (hunk 1 above)
Comment (chandelieraxel) [reply migrations/20190605091052-Table-Keyword_With_Fk.js]: Done

**Pull request merged**